xrpl/wallet/main.py

```diff
@@ -47,7 +46,10 @@ def __str__(self: Wallet) -> str:
         Returns:
             A string representation of a Wallet.
         """
-        return (
-            f"seed: {self.seed}\npub_key: {self.pub_key}\n"
-            f"priv_key: {self.priv_key}\nclassic_address: {self.classic_address}\n"
+        return "\n".join(
+            [
+                f"pub_key: {self.pub_key}",
+                "priv_key: -HIDDEN-",
```

Comment:
felt like a good idea not to print this, in the case users are turning this into a log

Reply:
rippled does something similar in logs, but I forget what magic string it puts instead. It's `REDACTED` or something.